Tasks/CondaEnvironmentV1/Tests/L0_conda.ts

```diff
@@ -147,17 +147,22 @@ it('fails if `conda` is not found', async function (done: MochaDone) {
         updateConda: false
     };
 
+    // Can't use `assert.throws` with an async function
+    // Node 10: use `assert.rejects`
+    let error: any | undefined;
     try {
         await uut.condaEnvironment(parameters, Platform.Windows);
-        done(new Error('should not have succeeded'));
```

Comment:
I was using both patterns before.  I believe this way is better: failing tests in the old pattern (using `done`) would simply give the "timeout of 2000 ms has been reached, make sure you are calling done()" message.